### PR TITLE
Removing hard coded stats url from build-env script and passing urls to grid proxy and graphql client

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Removing hard coded stats url from build-env script and passing urls to grid proxy and graphql client

### Changes
-  passing urls to grid proxy and graphql client instead of network
- In playground passed the appropriate url as a parameter to both grid proxy and graphql client , instead of passing the network and urls if set
- In new stats passed networks as string array instead of network enum
-  Adding relayUrl to GridClientConfig
- Updating config file in playground's public folder

### Related Issues

#2267
#2329
### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/new_stats/src/components/network_filter.vue [hunk 1]
```diff
@@ -85,12 +84,12 @@ const networks = ref([
 ]);
 
 // initial state; all network selected
-const selectedNetworks = ref([Network.Dev, Network.Main, Network.Test]);
+const selectedNetworks = ref(["dev", "main", "test"]);
```
Comment (AlaaElattar) [inline packages/new_stats/src/components/network_filter.vue:87]: why u didn't add `qa` ?

Comment (maayarosama) [reply packages/new_stats/src/components/network_filter.vue:87]: I'm not entirely sure why qa wasn't added from the start, @0oM4R do you have any idea why?

Comment (0oM4R) [reply packages/new_stats/src/components/network_filter.vue:87]: Yes we don't have to include qa net on our stats, the required network is only dev, test and main

Comment (AlaaElattar) [reply packages/new_stats/src/components/network_filter.vue:87]: Thank youu, i didn't know that.

**Review by AlaaElattar — APPROVED** [review]

**Pull request merged**